assignment-tool/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -11690,9 +11688,9 @@ private void initializeAssignment(SessionState state)
 			try {
 				st = SiteService.getSite((String) state.getAttribute(STATE_CONTEXT_STRING));
 				if (contentreviewSiteMin > 0 && st.getTitle().length() < contentreviewSiteMin){
-					addAlert(state, rb.getFormattedMessage("review.sitechars", new Object[]{contentreviewSiteMin, reviewServiceName}));
+					addAlert(state, rb.getFormattedMessage("review.sitechars", new Object[]{reviewServiceName, contentreviewSiteMin}));
```

Comment:
We do, the user should only see this warning if they explicitly select a
content review option.

-- 
  Matthew Buckett, VLE Developer, IT Services, University of Oxford